### ISSUE TITLE
Compatibility with Storybook 8

### ISSUE THREAD

**Issue by sapkra** [storybookjs/addon-svelte-csf#172]
### Describe the bug

I tried to upgrade to the newest beta of storybook but I'm getting the error below.

### Steps to reproduce the behavior

Install latest beta version of storybook together with [addon-svelte-csf](https://github.com/storybookjs/addon-svelte-csf)

### Screenshots and/or logs

```console
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @storybook/addon-svelte-csf@4.1.1
npm ERR! Found: @storybook/svelte@8.0.0-beta.1
npm ERR! node_modules/@storybook/svelte
npm ERR!   dev @storybook/svelte@"8.0.0-beta.1" from the root project
npm ERR!   @storybook/svelte@"8.0.0-beta.1" from @storybook/svelte-vite@8.0.0-beta.1
npm ERR!   node_modules/@storybook/svelte-vite
npm ERR!     @storybook/svelte-vite@"8.0.0-beta.1" from @storybook/sveltekit@8.0.0-beta.1
npm ERR!     node_modules/@storybook/sveltekit
npm ERR!       dev @storybook/sveltekit@"8.0.0-beta.1" from the root project
npm ERR!   1 more (@storybook/sveltekit)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @storybook/svelte@"^7.0.0" from @storybook/addon-svelte-csf@4.1.1
npm ERR! node_modules/@storybook/addon-svelte-csf
npm ERR!   dev @storybook/addon-svelte-csf@"4.1.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @storybook/svelte@7.6.13
npm ERR! node_modules/@storybook/svelte
npm ERR!   peer @storybook/svelte@"^7.0.0" from @storybook/addon-svelte-csf@4.1.1
npm ERR!   node_modules/@storybook/addon-svelte-csf
npm ERR!     dev @storybook/addon-svelte-csf@"4.1.1" from the root project
```

**Comment by shilman** [comment]
:rocket: Issue was released in [`v4.1.2-next.0`](https://github.com/storybookjs/addon-svelte-csf/releases/tag/v4.1.2-next.0) :rocket:

**Comment by JReinhold** [comment]
@sapkra please try out version `4.1.2-next.0` to see if it works in your Storybook 8 codebase and let me know if anything is broken.

**Comment by sapkra** [comment]
Thank you, the error seems to be resolved.

**Comment by shilman** [comment]
:rocket: Issue was released in [`v4.1.2`](https://github.com/storybookjs/addon-svelte-csf/releases/tag/v4.1.2) :rocket:
